examples/MQ/pixelDetector/run/scripts/test-pixelSim.sh.in

```diff
@@ -51,37 +51,44 @@ dds-topology -s $DDS_SESSION_ID --activate $FAIRMQ_DDS_TOPO_FILE --disable-valid
 
 sleep 1
 
-fairmq-dds-command-ui -w "IDLE"
-fairmq-dds-command-ui -c i
-fairmq-dds-command-ui -c k
-fairmq-dds-command-ui -c b
-fairmq-dds-command-ui -c x
-fairmq-dds-command-ui -c j
-fairmq-dds-command-ui -c r
+for cmd in IDLE i k b x j r
+do
+	echo ""
+	if (( ${#cmd} == 1 ))
+	then
+		echo "***** fairmq-dds-command-ui -c $cmd"
+		fairmq-dds-command-ui -c "$cmd"
+	else
+		echo "***** Waiting for state $cmd"
+		time fairmq-dds-command-ui -w "$cmd"
+	fi
+done
 
 samplers_string="main/SamplerGroup.*"
-echo "waiting for $samplers_string to transition RUNNING->READY" 
-fairmq-dds-command-ui -w "RUNNING->READY" -p $samplers_string
+echo ""
+echo "***** waiting for $samplers_string to transition RUNNING->READY" 
+time fairmq-dds-command-ui -w "RUNNING->READY" -p $samplers_string
 echo "...$samplers_string are READY, sending shutdown..."
 
 sleep 2
 
-echo "check dds states again"
+echo ""
+echo "***** check dds states again"
 fairmq-dds-command-ui -c c
 echo "^ ^ ^ ^ ^ ^ ^ ^ ^ ^ ^ ^"
 
-fairmq-dds-command-ui -c s
-sleep 1
-fairmq-dds-command-ui -c t
-sleep 1
-fairmq-dds-command-ui -c d
-sleep 1
-fairmq-dds-command-ui -c q
-sleep 1
+for cmd in s t d q
+do
+	echo ""
+	echo "***** fairmq-dds-command-ui -c $cmd"
```

Comment:
Isn't `set -x` doing what you are you doing here?

Reply:
Yes, `set -x` would do that. But do we want the whole other test logic to spam the logs?
What about rephrasing the output? Like "Sending command $cmd"?

Reply:
```
set -x
fairmq-dds-command-ui -c s
fairmq-dds-command-ui -c t
fairmq-dds-command-ui -c d
fairmq-dds-command-ui -c q
set +x
```
What about this?

I am honestly not sure if the loop is increasing readability here.